index.html

```diff
@@ -0,0 +1,159 @@
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+   </div>
+   <hr class="line-3">
+
+   <ul class="lists">
+      <li class="Lan">Language 
+          <ul class="lan-list">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+      </li>
+        
+      <li class="Frame"> Framework
+          <ul class="Frame-list">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
```

Comment:
- [ ] In this section, the listed languages' color is slightly different from the header. Kindly adjust that to bring out the contrast. And also kindly adjust the spacing to match the original design.

![image](https://user-images.githubusercontent.com/98453979/213499429-f7c1d08f-ae76-4788-9f36-fb7a6b25c8f9.png)

#### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/213499609-4ecd3e7f-6b74-4c7f-8e90-45dcdf02ab51.png)